src/app/examples/todos/todos.reducer.spec.ts

```diff
@@ -0,0 +1,67 @@
+import * as fromTodo from './todos.reducer';
```

Comment:
I saw this approach around the internet but would preffer if we stayed consistent with the stile used around this project which is to import things explicitly like `import { TodosState } from './todos.reducer'` (ofcourse all the used thing in a single import). Thank you!